instrumentation/thrift/javaagent/build.gradle.kts

```
@@ -0,0 +1,17 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.apache.thrift")
+    module.set("apache-thrift")
+    versions.set("0.14.1")
+  }
+}
+
+dependencies {
+  implementation("org.apache.thrift:libthrift:0.14.1")
```

`implementation` will result in the library being included inside the javaagent, which probably is not what you're aiming for. You can use `library` instead (our own "syntactic sugar" configuration that's equivalent to `compileOnly`):
```suggestion
  library("org.apache.thrift:libthrift:0.14.1")
```